### PR TITLE
Update and move core uninstall article to install area

### LINKED ISSUE #16062: Dupe content
@Thraka while reviewing a PR tonight, I realized that the content in this article is duplicated of what we already had in https://docs.microsoft.com/en-us/dotnet/core/versions/remove-runtime-sdk-versions?tabs=windows#determine-what-is-installed. We should decide what to do about this.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7f2842ce-ee53-ec35-90c6-791a952dfc3b
* Version Independent ID: 79ec96d5-9c2c-1eaa-feb8-40b0afee6a30
* Content: [Check installed .NET Core versions on Windows, Linux, and macOS - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/install/how-to-detect-installed-versions)
* Content Source: [docs/core/install/how-to-detect-installed-versions.md](https://github.com/dotnet/docs/blob/master/docs/core/install/how-to-detect-installed-versions.md)
* Product: **dotnet-core**
* GitHub Login: @Thraka
* Microsoft Alias: **adegeo**

### PR DESCRIPTION
**Pull request by Thraka**
## Summary

- Moved article from **core/versions** to **core/install**.
- Added redirect.
- Updated TOC.
- Updated screenshot to win 10.
- Changed tabs to pivots.

Acro score is 92

[Internal preview link](https://review.docs.microsoft.com/en-us/dotnet/core/install/remove-runtime-sdk-versions?branch=pr-en-us-18129)

Fixes #16062

### CODE REVIEW

File: docs/core/install/remove-runtime-sdk-versions.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+Over time, as you install updated versions of the .NET Core runtime and SDK, you may want to remove outdated versions of .NET Core from your machine. Removing older versions of the runtime may change the runtime chosen to run shared framework applications, as detailed in the article on [.NET Core version selection](../versions/selection.md).
+
+## Should I remove a version?
+
+The [.NET Core version selection](../versions/selection.md) behaviors and the runtime compatibility of .NET Core across updates enables safe removal of previous versions. .NET Core runtime updates are compatible within a major version 'band' such as 1.x and 2.x. Additionally, newer releases of the .NET Core SDK generally maintain the ability to build applications that target previous versions of the runtime in a compatible manner.
+
+In general, you only need the latest SDK and latest patch version of the runtimes required for your application. Instances where keeping older SDK or Runtime versions include maintaining *project.json*-based applications. Unless your application has specific reasons for earlier SDKs or runtimes, you may safely remove older versions.
```
Comment (tdykstra) [inline docs/core/install/remove-runtime-sdk-versions.md:18]:
```suggestion
In general, you only need the latest SDK and latest patch version of the runtimes required for your application. Instances where you might want to keep older SDK or runtime versions include maintaining *project.json*-based applications. Unless your application has specific reasons for earlier SDKs or runtimes, you may safely remove older versions.
```

File: docs/core/install/remove-runtime-sdk-versions.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+Select any versions you want to remove from your machine and click **Uninstall**.
+
+::: zone-end
+
+::: zone pivot="os-linux"
+
+There are more options to uninstall .NET Core (either SDK or runtime) on Linux. The best way for you to uninstall .NET Core is to mirror the action you used to install .NET Core. The specifics depend on your chosen distribution and the installation method.
+
+> [!IMPORTANT]
+> For Red Hat installations, consult the [Red Hat Getting Started Guide](https://access.redhat.com/documentation/en-us/net_core/2.0/html/getting_started_guide/gs_install_dotnet#install_register_rehel) for information on installing and uninstalling .NET Core.
```
Comment (tdykstra) [inline docs/core/install/remove-runtime-sdk-versions.md:41]: it's not a MS link, but will it work if you drop the /en-us?

File: docs/core/install/remove-runtime-sdk-versions.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+Starting with .NET Core 2.1, there's no need to uninstall the .NET Core SDK when upgrading it using a package manager. The package manager `update` or `refresh` commands will automatically remove the older version upon the successful installation of a newer version.
+
+If you installed .NET Core using a package manager, you use that same package manager to uninstall .NET SDK or runtime. .NET Core installations support most popular package managers. Consult the documentation for your distribution's package manager for the precise syntax on your environment:
```
Comment (tdykstra) [inline docs/core/install/remove-runtime-sdk-versions.md:45]:
```suggestion
If you installed .NET Core using a package manager, you use that same package manager to uninstall .NET SDK or runtime. .NET Core installations support most popular package managers. Consult the documentation for your distribution's package manager for the precise syntax in your environment:
```

File: docs/core/install/remove-runtime-sdk-versions.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+```bash
+apt-get remove dotnet-host
+```
+
+Note that there's no version attached to `dotnet-host`.
+
+If you installed using a tarball, you must remove .NET Core using the manual method.
+
+On Linux, you must remove the SDKs and runtimes separately, by removing the versioned directories. To be clear, this deletes the SDK and runtime from disk. For example, to remove the 1.0.1 SDK and runtime, you would use the following bash commands:
```
Comment (tdykstra) [inline docs/core/install/remove-runtime-sdk-versions.md:68]: Could omit "To be clear"

File: docs/core/install/remove-runtime-sdk-versions.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+sudo rm -rf /usr/local/share/dotnet/host/fxr/$version
+```
+
+The parent directories for the SDK and runtime are listed in the output from the `dotnet --list-sdks` and `dotnet --list-runtimes` command, as shown in the earlier table.
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+On Mac, you must remove the SDKs and runtimes separately, by removing the versioned directories. To be clear, this deletes the SDK and runtime from disk. For example, to remove the 1.0.1 SDK and runtime, you would use the following bash commands:
```
Comment (tdykstra) [inline docs/core/install/remove-runtime-sdk-versions.md:85]: Could omit "To be clear"

File: docs/core/install/remove-runtime-sdk-versions.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+Starting with Visual Studio 2019 version 16.3, Visual Studio is in charge of its own copy of the .NET Core SDK. For that reason, you no longer see those SDK versions in the **Apps & features** dialog.
+
+## Remove the NuGet fallback folder
+
+Before .NET Core 3.0 SDK, the .NET Core SDK installers used a folder named *NuGetFallbackFolder* to store a cache of NuGet packages. This cache was used during operations such as `dotnet restore` or `dotnet build /t:Restore`. The *NuGetFallbackFolder* is located at *C:\Program Files\dotnet\sdk* on Windows and at */usr/local/share/dotnet/sdk* on macOS.
+
+You may want to remove this folder, if:
+
+- You're only developing using .NET Core 3.0 SDK or later versions.
+- You're developing using .NET Core SDK versions earlier than 3.0, but you can work online and things can be slower once.
```
Comment (tdykstra) [inline docs/core/install/remove-runtime-sdk-versions.md:125]: "things can be slower once" isn't very clear

**Review by tdykstra — APPROVED** [review]
Looks great, a few minor suggestions.

File: docs/core/install/remove-runtime-sdk-versions.md (hunk 2 above)
Comment (Thraka) [reply docs/core/install/remove-runtime-sdk-versions.md:41]: No, it doesn't work. It seems on the language dropdown on the page, only English is available.

**Pull request merged**